pkg/controller/kabaneroplatform/landing.go

```diff
@@ -120,6 +120,9 @@ func deployLandingPage(k *kabanerov1alpha1.Kabanero, c client.Client) error {
 		}
 
 		apiUrl, err := url.Parse(apiUrlString)
+		if len(apiUrl.Scheme) == 0 {
```

Comment:
Same comment here, we need to check err first before calling `len(apiUrl.Scheme)`